test/models/timeoutable_test.rb

```diff
@@ -50,4 +50,10 @@ class TimeoutableTest < ActiveSupport::TestCase
     user = create_admin(remember_created_at: Time.current)
     assert user.timedout?(31.minutes.ago)
   end
+
+  test 'should return false if rememberable is enabled and has not expired' do
+    user = create_user
+    user.remember_me!
+    refute user.timedout?(31.minutes.ago)
+  end
 end
```

Comment:
We could add one more test to cover the scenario that the rememberable is enabled and is expired.